interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/SubprocessComponentDefinitionExtractor.scala

```diff
@@ -46,29 +46,34 @@ abstract class SubprocessDefinitionExtractor {
 
 }
 
+sealed trait FragmentDefinitionError
+
+case object EmptyFragmentError extends FragmentDefinitionError
```

Comment:
what message is propaged to UI when we have this kind of error?

Reply:
Ok, I guess I've found it, it is mapped to `InvalidSubprocess` so we have description: `Node $nodeId uses fragment $id which is invalid` 

What do you think about a change, so we have:
`InvalidSubprocess(id: String, nodeId: String, cause: FragmentDefinitionError)` and have dedicated error description for each kind of `cause`?

Reply:
IMO exact errors should be presented when smb is in fragment. When fragment is used, user should rather don't think about exact problems - after all, fragments are for decomposition